### ISSUE TITLE
Release with darwin/arm64 Support

### ISSUE THREAD

**Issue by bflad** [hashicorp/terraform-provider-vault#1038]
### Terraform Version

```
Terraform v0.15.1
```

### Affected Resource(s)

N/A

### Terraform Configuration Files

N/A

### Debug Output

N/A

### Expected Behavior

Able to install provider on `darwin/arm64` platform without manually compiling.

### Actual Behavior

Error during `terraform init`:

```
Error: Incompatible provider version

Provider registry.terraform.io/hashicorp/vault v#.#.# does not have a package
available for your current platform, darwin_arm64.

Provider releases are separate from Terraform CLI releases, so not all
providers are available for all platforms. Other versions of this provider may
have different platforms supported.
```

Need to manually compile and install provider for `darwin/arm64` support.

### Steps to Reproduce

1. `terraform init`

### Important Factoids

All that is generally required is building and testing the provider using Go 1.16, e.g. updating the `.go-version` file. Terraform Providers are generally not Go feature intensive or used as an upstream Go package, so Go updates are relatively low risk. 

We are looking to enable `darwin/arm64` releases in Terraform CLI soon.

### References

- https://github.com/hashicorp/terraform/issues/27257

**Comment by bodinsamuel** [comment]
Hello, thanks for raising and fixing the issue, however no releases has been done since this was fixed. 
Is there any other issue? or should we expect a release soon? ☺️

**Comment by jasonodonnell** [comment]
We aim to release 3rd Wednesday of each month, so likely next week @bodinsamuel.

**Comment by jasoares** [comment]
Just for clarity this was actually released in [v2.21.0](https://github.com/hashicorp/terraform-provider-vault/releases/tag/v2.21.0) 3 days ago
